src/domUpdates.js

```diff
@@ -99,45 +102,44 @@ let domUpdates = {
     if (event.target.classList.contains('favorite')) {
       this.favoriteCard(event, recipeRepo, user);
     } else if (event.target.classList.contains('card-picture')) {
-      this.displayDirections(event, recipeRepo);
+      this.displayDirections(event, recipeRepo, this.ingredientsDataDOM);
     } else if (event.target.classList.contains('home')) {
       favButton.innerHTML = 'View Favorites';
       this.populateCards(recipeRepo.recipes);
     }
   },
 
 
-  displayDirections(event, recipeRepo) {
-    const matchingRecipe = recipeRepo.recipes.find(recipe => {
-      if (recipe.id === parseInt(event.target.id)) {
-        return recipe;
-      }
-      return recipe;
+  displayDirections(event, recipeRepo, ingredientsDataDOM) {
```

Comment:
Awesome job getting that data from the fetch request to work in the class parameter!